### PR TITLE
Change date.humanize.today to 'Today at {time}'

### PR DESCRIPTION
**Pull request by toyelowo**
## Description

Humanize future 'today' date as `Today at {time}`

## Type of change

- [x] **react-i18n** - Major: Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have added a changelog entry, prefixed by the type of change noted above

### CODE REVIEW

File: packages/react-i18n/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [3.1.0] - 2020-05-25
```
Comment (alexandcote) [inline packages/react-i18n/CHANGELOG.md:8]:
```suggestion
<!-- ## [Unreleased] -->
```

File: packages/react-i18n/src/i18n.ts [hunk 2]
```diff
@@ -474,7 +474,7 @@ export class I18n {
     }).toLocaleLowerCase();
 
     if (isToday(date)) {
-      return time;
+      return `Today at ${time}`;
```
Comment (alexandcote) [inline packages/react-i18n/src/i18n.ts:477]: The package should already do this for you. Maybe I can help you configure the package if you want.

Comment (toyelowo) [reply packages/react-i18n/src/i18n.ts:477]: Yeah that'll be awesome. Thanks, I'll close the PR then

Comment (alexandcote) [inline packages/react-i18n/src/i18n.ts:477]: We should not hard code the string here. We should use the translation file to translate this into multiple languages. For example, in French, the text should be `Aujourd'hui à ${time}`. 

Updating this is considered a breaking change because the consumer needs to add a new translation key. I'm wondering if there is a reason why we didn't add this version in the first place. Maybe @TheMallen has more context since he works more than me on this package.

File: yarn.lock [hunk 3]
```diff
@@ -11410,6 +11410,11 @@ typescript@^3.7.5:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-3.7.5.tgz#0692e21f65fd4108b9330238aac11dd2e177a1ae"
   integrity sha512-/P5lkRXkWHNAbcJIiHPfRoKqyd7bsyCma1hZNUGfn20qm64T6ZBlrzprymeu918H+mB/0rIg2gGK/BXkhhYgBw==
 
+ua-parser-js@^0.7.17:
```
Comment (alexandcote) [inline yarn.lock:11413]: I think you can revert this change.

File: packages/react-i18n/CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+<!-- ## [Unreleased] -->
+
+### Changed
+
+- Updated `I18n#humanizeDate` to humanize today's date as `Today at {time}` [[#1459](https://github.com/Shopify/quilt/pull/1459)]
```
Comment (alexandcote) [inline packages/react-i18n/CHANGELOG.md:12]: Can you add the link to the migration guide like above 😃

File: packages/react-i18n/src/i18n.ts [hunk 5]
```diff
@@ -518,7 +518,7 @@ export class I18n {
     }).toLocaleLowerCase();
 
     if (isToday(date)) {
-      return time;
+      return this.translate('date.humanize.today', {time});
```
Comment (alexandcote) [inline packages/react-i18n/src/i18n.ts:521]: 💯

File: packages/react-i18n/migration-guide.md [hunk 6]
```diff
@@ -2,6 +2,19 @@
...
+
+```json
+"date": {
+  "humanize": {
+    ...
+    "today": "Today at {time}"
+  }
+},
+```
+
```
Comment (jaxee) [inline packages/react-i18n/migration-guide.md:17]: ✨ so good
